### PR TITLE
Add WORKObjects-related WMS layer to map

### PR DESCRIPTION
**Pull request by monodo**

### CODE REVIEW

**Comment by monodo** [comment]
@nicolas-sitylb please review

File: static/customWidgets/GeometryWidget/geometrywidget.js [hunk 1]
```diff
@@ -35,6 +34,35 @@
...
+    ;
+
+    var layers =  this.options.wms_layers.split(',');
+    var ol_layers = [];
+    for (var i=0; i < layers.length; i++) {
+        ol_layers.push(
+            new ol.layer.Image({
+            source: new ol.source.ImageWMS({
+              url: layers[i],
+              projection: "EPSG:2056",
```
Comment (nicolas-sitylb) [inline static/customWidgets/GeometryWidget/geometrywidget.js]: What happens if a layer in the groups comes with an other projection system? Maybe it's more robust to parse and extract the CRS parameter from the WMS URL?

File: static/customWidgets/GeometryWidget/geometrywidget.js [hunk 2]
```diff
@@ -34,33 +34,22 @@
...
-            opacity: 1,
-          })
+    var layer_list = [];
+
+    this.options.wms_layers.split(",").forEach(function (item, index) {
+      layer_list.push(
+        new ol.layer.Image({
+          source: new ol.source.ImageWMS({
+            url: item,
+            projection: "EPSG:2056",
```
Comment (nicolas-sitylb) [inline static/customWidgets/GeometryWidget/geometrywidget.js:44]: Use CRS parameter from WMS here?

**Review by nicolas-sitylb — APPROVED** [review]

File: static/customWidgets/GeometryWidget/geometrywidget.js (hunk 2 above)
Comment (monodo) [reply static/customWidgets/GeometryWidget/geometrywidget.js:44]: => The whole widget is for EPSG:2056

Comment (nicolas-sitylb) [reply static/customWidgets/GeometryWidget/geometrywidget.js:44]: Then what if people want to upload WGS84 KML files? Or WorldMercator GPX data? Maybe we should consider that in the future as well?

Comment (monodo) [reply static/customWidgets/GeometryWidget/geometrywidget.js:44]: => KML are reprojected on import => https://github.com/yverdon/geocity/blob/master/static/customWidgets/GeometryWidget/geometrywidget.js#L308

**Pull request merged**